src/gobexport/exporter/config/brk.py

```diff
@@ -42,7 +42,7 @@
 
 
 def _get_filename_date():
-    meta = requests.get(f"{get_host()}/gob/brk/meta/1").json()
+    meta = requests.get(f"{get_host()}/gob/secure/brk/meta/1").json()
```

Comment:
Gaat dit werken als brk een secure catalog wordt? Of kun je het niet gewoon opvragen via de requests module zodra de header automatisch wordt toegevoegd?